google-cloud-contrib/pom.xml

```diff
@@ -23,7 +23,7 @@
   <dependencies>
     <dependency>
       <groupId>${project.groupId}</groupId>
-      <artifactId>google-cloud</artifactId>
+      <artifactId>google-cloud-storage</artifactId>
```

Comment:
I believe we can get rid of this dependency, no need to import anything here, we do that in child modules. Makes sense?